src/frontend/src/lib/components/settings/Settings.svelte

```diff
@@ -44,9 +44,7 @@
 	};
 
 	$: {
-		if ($authSignedIn && POUH_ENABLED) {
-			loadUserProfile({ identity });
-		}
+		emit({ message: 'oisyRefreshUserProfile' });
```

Comment:
- Do we actually need refresh? Settings isn't wrapper with the loader component?
 - If we have to keep "refresh", the we can still maybe reload only if `POUH_ENABLED`?

Reply:
you are right, it does not need necessarily a refresh, removed!